streams/src/test/java/org/apache/kafka/streams/processor/internals/InternalTopologyBuilderTest.java

```diff
@@ -780,4 +782,17 @@ public void shouldNotAllowToAddGlobalStoreWithSourceNameEqualsProcessorName() {
             sameNameForSourceAndProcessor,
             new MockProcessorSupplier());
     }
+
+    @Test
+    public void shouldAllowAddGlobalStoreWithNonKVStore() {
+        final StoreBuilder windowStoreBuilder = new MockWindowStoreBuilder("store", false);
```

Comment:
Do we need `MockWindowStoreBuilder` and `MockWindowStore` ? Can't we just use `Mockito` ?

Reply:
We could use `Mockito`. I went for EasyMock as there were already `MockKeyValueStoreBuilder` and `MockKeyValueStore`, so it seemed like a nice fit, at least for someone who, like me, is only getting started with the Kafka code. I since saw discussions about using Mockito instead of EasyMock on the mailing list, and I agree that it makes sense to use Mockito.

Reply:
Actually I can't seem to find any use of mockito anywhere in this repo, so perhaps it is better to stick with easymock for this PR? What do you think?